src/routes/safe/components/Transactions/TxList/TxInfoTransfer.tsx

```diff
@@ -18,13 +19,29 @@ export const TxInfoTransfer = ({ txInfo }: { txInfo: Transfer }): ReactElement |
     if (assetInfo && assetInfo.type === 'Transfer') {
       if (txInfo.direction.toUpperCase() === 'INCOMING') {
         setDetails({
-          title: `Received ${assetInfo.amountWithSymbol} from:`,
+          title: (
+            <Text size="xl" as="span">
```

Comment:
Does this intefere with anything? Or is this used where I made the previous comment about sending components as props?

Reply:
That's correct. Because I want to set the middle portion of the `title` bold I can't send a simple string.